module2/homework/maxOfVector/maxOfVector.hpp

```diff
@@ -3,6 +3,14 @@
 #include <limits>
 
 int maxOfVector(const std::vector<int>& vec) {
-    // TODO: Implement me :)
-    return {};
-}
+    
+    int maxVec = vec[0];
+
+    for (size_t i = 0; i < vec.size(); ++i) {
+        if (maxVec <= vec[i]) {
```

Comment:
It will be enough when we check if `maxVec` is smaller than `vec[i]`: `maxVec < vec[i]`. When it is equal it can remain the same. It will not go into `if` unnecessary

Reply:
I agree with you.